web/server/web_client.c

```diff
@@ -1266,7 +1266,7 @@ static inline void web_client_send_http_header(struct web_client *w) {
                 while((bytes = SSL_write(w->ssl.conn, buffer_tostring(w->response.header_output), buffer_strlen(w->response.header_output))) < 0) {
                     count++;
                     if(count > 100 || (errno != EAGAIN && errno != EWOULDBLOCK)) {
-                        error("Cannot send HTTP headers to web client.");
+                        error("Cannot send HTTPS headers to web client.");
```

Comment:
Hi @amoss ,

I will give a suggestion for you, OpenSSL can give you more details about the error as you can see in the documentation (https://netdata-cloud.slack.com/), please, call SSL_get_error() together with ERR_error_string(), for an example you can see https://github.com/netdata/netdata/blob/master/backends/backends.c#L1022 .

Reply:
This is a good point but I don't want to complicate this issue by changing the code being tested as well. In the next sprint I'll be rewriting the URL parsing and then I can take a look at the error codes.

Reply:
All right!